### ISSUE TITLE
Visitor.VisitFunc() receiving method, the value of operationId is not good

### ISSUE THREAD

**Issue by podhmo** [podhmo/reflect-openapi#24]
e.g. 

```
  "paths": {
    "/todos": {
      "get": {
        "operationId": "main.(*Store).ListTodo-fm",
        "responses": {
          "200": {
            "content": {
              "application/json": {
                "schema": {
                  "items": {
                    "$ref": "#/components/schemas/Todo"
                  },
                  "type": "array"
                }
              }
            },
            "description": ""
          },
          "default": {
            "description": ""
          }
       ```

**Comment by podhmo** [comment]
hmm.. `"main.(*Store).ListTodo-fm"`

**Comment by podhmo** [comment]
"main.Store.ListTodo" ?

- `<package name>.<function name>`
- `<package name>.<struct name>.<method name>`

**Comment by podhmo** [comment]
- #93

**Comment by podhmo** [comment]
- #102

**Comment by podhmo** [comment]
fixed by use reflect-shape v0.4
